src/main/java/de/teamlapen/vampirism/client/gui/VampirismHUDOverlay.java

```diff
@@ -55,6 +57,7 @@
 
     private final Minecraft mc;
     private final ResourceLocation icons = new ResourceLocation(REFERENCE.MODID + ":textures/gui/icons.png");
+    private final ResourceLocation icons_poison = new ResourceLocation(REFERENCE.MODID, "textures/gui/icons_poison.png");
```

Comment:
If you want to use the original item, please bind the MC texture or add the icon to our custom file.